packages/samples/headless-react/src/pages/DependentFacetPage.tsx

```diff
@@ -1,15 +1,14 @@
 import {
```

Comment:
I pretty much completely changed this doc page since we can no longer subscribe to the controller from outside the component.

IMO, it's easier to read as documentation now, since everything is in one single file.